terra-jupyter-base/jupyter_notebook_config.py

```diff
@@ -0,0 +1,51 @@
+# adapted from https://github.com/jupyter/docker-stacks/blob/master/base-notebook/jupyter_notebook_config.py
+
+from jupyter_core.paths import jupyter_data_dir
+import subprocess
+import os
+import errno
+import stat
+
+c = get_config()
+c.NotebookApp.ip = '0.0.0.0'
```

Comment:
To get this to work on a Mac I had to change `localhost` to `0.0.0.0`. Hopefully that does not cause problems for real clusters -- will verify.